RecoParticleFlow/PFProducer/python/particleFlowBlock_cfi.py

```diff
@@ -160,37 +164,26 @@ def _findIndicesByModule(name):
             ret.append(i)
    return ret
 
-
-from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+#
 # kill tracks in the HGCal
-_insertGeneralTracksImporter = {}
-for idx in _findIndicesByModule('GeneralTracksImporter'):
-    _insertGeneralTracksImporter[idx] = dict(
-        importerName = cms.string('GeneralTracksImporterWithVeto'),
-        veto = cms.InputTag('hgcalTrackCollection:TracksInHGCal')
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+_insertTrackImportersWithVeto = {}
+_trackImporters = ['GeneralTracksImporter','ConvBremTrackImporter',
+                   'ConversionTrackImporter','NuclearInteractionTrackImporter']
+for importer in _trackImporters:
+  for idx in _findIndicesByModule(importer):
+    _insertTrackImportersWithVeto[idx] = dict(
+      vetoEndcap = cms.bool(True),
```

Comment:
```suggestion
      vetoEndcap = True,
```
the parameter is supposedly defined earlier